labsync/sync_prepare.sh

```diff
@@ -49,7 +49,7 @@ hub checkout master
 
 # maybe some errors happened last time, the branch wont be clean
 is_clean="`hub status |grep clean`"
```

Comment:
so is_clean is not used anymore.

Reply:
Good catch, `is_modified` is same issue.

Reply:
Sorry, forgot delete it.